x-pack/plugins/cases/public/components/all_cases/selector_modal/all_cases_selector_modal.tsx

```diff
@@ -53,23 +55,25 @@ export const AllCasesSelectorModal = React.memo<AllCasesSelectorModalProps>(
     );
 
     return isModalOpen ? (
-      <Modal onClose={closeModal} data-test-subj="all-cases-modal">
-        <EuiModalHeader>
-          <EuiModalHeaderTitle>{i18n.SELECT_CASE_TITLE}</EuiModalHeaderTitle>
-        </EuiModalHeader>
-        <EuiModalBody>
-          <AllCasesList
-            hiddenStatuses={hiddenStatuses}
-            isSelectorView={true}
-            onRowClick={onClick}
-          />
-        </EuiModalBody>
-        <EuiModalFooter>
-          <EuiButton color="text" onClick={closeModal}>
-            {i18n.CANCEL}
-          </EuiButton>
-        </EuiModalFooter>
-      </Modal>
+      <QueryClientProvider client={casesQueryClient}>
```

Comment:
This will enable the cases selector modal to use react-query hooks. It doesn't affect the bundle size.